changelog/unreleased/issue-4004

```diff
@@ -0,0 +1,12 @@
+Bugfix: Allow use of container level SAS/SAT tokens with Azure backend.
```

Comment:
The summary must not end with a punctuation character.

Reply:
Easy, I'll remove